src/pudl/output/ferc1.py

```diff
@@ -1191,7 +1191,22 @@ def _out_ferc1__explosion_tags(table_dimensions_ferc1) -> pd.DataFrame:
         .reset_index()
         .drop(columns=["notes"])
     )
-    return tags_all
+    # Add the correction records to the tags with the same tags as the parent
+    idx = list(NodeId._fields)
```

Comment:
I think if we're re-using this `list(NodeId._fields)` in a bunch of places, including line 1187, we might as well give it a meaningful name (`set_index(idx)` doesn't really tell you more than `set_index(adfadfadfa)`).

This could be `node_id_fields = list(NodeId._fields)` maybe?

Reply:
ah yes you are so right `idx` is a completely meaningless name. this `list(NodeId._fields)` is used 13 (!) times throughout this module. In this context it's really just in index columns or primary key columns of the tags. so maybe `tag_idx`? in other contexts its used as the pk/idx of a calculation component or to identify a node in the calculation forest. but in the tag space we aren't really working with networks and so caling them node_id's feels a little wrong.